app/Http/Controllers/Auth/LoginController.php

```diff
@@ -17,9 +18,11 @@ class LoginController extends Controller
     | redirecting them to your home screen. The controller uses a trait
     | to conveniently provide its functionality to your applications.
     |
-    */
+     */
```

Comment:
```suggestion
    */
```
Sao thêm dấu cách đây chi?